charts/sn-platform/templates/presto/presto-jmx-exporter-configmap.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+{{- if and .Values.components.sql_worker .Values.presto.exporter.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-presto-jmx-exporter"
+  namespace: {{ template "pulsar.namespace" . }}
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+data:
+  jmx_exporter.yaml: |
+    startDelaySeconds: 0
+    jmxUrl: service:jmx:rmi://localhost:9081/jndi/rmi://localhost:9081/jmxrmi
+    ssl: false
+    lowercaseOutputName: false
+    lowercaseOutputLabelNames: false
+    whitelistObjectNames:
+{{- with .Values.presto.exporter.whitelist }}
+{{- toYaml . | nindent 4 }}
```

Comment:
seems the default in value is empty, that means no metrics get exported? probably better use blacklist?